dowhy/timeseries/temporal_shift.py

```diff
@@ -3,17 +3,16 @@
 from typing import List, Tuple
 
 def find_lagged_parent_nodes(graph:nx.DiGraph, node:str) -> Tuple[List[str], List[int]]:
```

Comment:
is this only for parents or for all predecessors? probably change the name to `find_lagged_predecessors`.

Reply:
DiGraph.predecessors(n)[[source]](https://networkx.org/documentation/stable/_modules/networkx/classes/digraph.html#DiGraph.predecessors)
Returns an iterator over predecessor nodes of n.

A predecessor of n is a node m such that there exists a directed edge from m to n.

According to their documentation, a predecessor refers to only parents in the graph